### DISCUSSION TITLE
Upgrade Digits OTP login

### DISCUSSION THREAD

**Discussion by Condize** [inspireui/support#5257] [Working on Features]
Please support this 
https://codecanyon.net/item/digits-wordpress-mobile-number-signup-and-login/19801105

for seamless integration between site and app

**Comment by Uhtredtb** [comment]
https://github.com/inspireui/support/discussions/5251

**Reply by saeed-golshan** [reply]
Is there any library to use digit for otp login?

**Reply by Uhtredtb** [reply]
> Is there any library to use digit for otp login?

Right now, I don't think so, still waiting for Inspire UI to check this feature out. 

Please kindly support this plugin.

**Comment by thesaluja** [comment]
This is a very good plugin. inspireui should integrate it.

**Reply by Uhtredtb** [reply]
Totally, agree please develop to work with the app.

**Reply by ecomdeveloperbd** [reply]
Highly recommend

**Comment by Yan-Aleman** [comment]
I will be looking forward to this integration very much. I really need it to avoid fake orders on my website. Also, all users want to register only through OTP. Digits is the best plugin ever.

**Comment by Condize** [comment]
My team have found a solution to integrate the app with digits and it works perfectly!

If any needs it, please contact me at admin(@)condize.com

**Reply by Uhtredtb** [reply]
The solution from Condize, is way to expensive which cost 200$++ and it not official from Inspire UI. Unless you would share back the source to this community so Inspire UI can develop it official??

Please let me know if that possible.

**Reply by genuinesourcecode** [reply]
This is very Good Plugins We also agreed

**Reply by hassanyousufx** [reply]
Yes, if you can make the code available to the public it would be really nice!

**Comment by tjain90** [comment]
Hi inspire ui team. Yes digit is very good. I support this feature.

**Comment by Uhtredtb** [comment]
@minhcasi Will you guys consider this feature to be compatible with your app??

**Comment by Yan-Aleman** [comment]
Hello.  What is the news on digits+firebase otp plugin compatibility?

**Comment by Uhtredtb** [comment]
Hi, can anyone please check and consider this??

It's 2022 now!

**Comment by ameenbadri7** [comment]
support this +1

**Comment by tryabp** [comment]
Login screen customisation is much needed in fluxstore. Digits is super awesome plugin. We are using it too. Fluxstore should set primary option to login as mobile OTP and Google / other social logins. Email + password login is anyway outdated. I hope @minhcasi and team works on this.

**Comment by HanielQC** [comment]
Hi all, This feature is available from version 2.10.5 and later. Cheer :)

Without purchase, you can view a demo App of your Website with the drag&drop FluxBuilder https://fluxbuilder.com/ for Free 🔻
🎯 Edit your App's UI by drag&drop
🎯 Build a Test APK to view how your App looks
Quick guide: https://www.notion.so/FluxBuilder-s-guide-a5164230ff444676b10b5cadc30302aa

Very Best!

**Comment by Yan-Aleman** [comment]
Please support like ticket #5351 (direct pay)

**Comment by naturaw-canine-food** [comment]
anyone using this?

mine just sends a code through firebase even if i have digits enabled .. and when we enter the code it says its wrong.
